### DISCUSSION TITLE
How to use subfinder to feed file containing list of domains

### DISCUSSION THREAD

**Discussion by vansh1** [projectdiscovery/subfinder#482] [Q&A] [answered]
How can i use subfinder with my file of domain I actually wanted to find the subdomains of subdomains how can we run subfinder with the file where my domain or subdomain stored

**Comment by ehsandeep** [comment, marked as answer]
@vansh1 subfinder can be used to discover subdomains for a given domain or list of domain names, to feed file as input containing a list of the domain, you can use `dL` or `list` flag with the latest version of subfinder.
